src/RandomDataGenerator.Tests/CityRandomizerTests.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Concurrent;
+using RandomDataGenerator.FieldOptions;
+using RandomDataGenerator.Randomizers;
+using Xunit.Abstractions;
+
+namespace RandomDataGenerator.Tests
+{
+	public class CityRandomizerTests
```

Comment:
It looks as if you used 8 spaces / 2 tabs ?

Can you change this so that the new code looks the same as other code in this project?

Reply:
I use VS2022 defaults, Tab size 4, Indent size 4, keep tabs. What is this projects text editor format preference?